src/main/protobuf/soc/moves.proto

```diff
@@ -0,0 +1,93 @@
+syntax = "proto2";
+
+package protos.soc;
+
+import "soc/inventory.proto";
+import "soc/board.proto";
+import "soc/state.proto";
+
+enum MoveType {
```

Comment:
@grogdotcom I already defined all of these here: https://github.com/SOC-Training-Tool/SOC-Server/blob/master/src/main/protobuf/game.proto#L59

Further, the making moves pattern is supported by the interface defined in this proto. Specifically, it all flows from the `TakeActionRequest` message:  https://github.com/SOC-Training-Tool/SOC-Server/blob/master/src/main/protobuf/game.proto#L213

I've documented the pattern in this table here:
https://docs.google.com/document/d/1koz0JKbiCvIWE3xHH8Ej4Wvs0iz5gztXkng8Bjg_StE/edit?usp=sharing
(Also found in the Catan AI document with more context).

This pattern works too, but I would encourage you to think how it plays out with action requests.

Also in general its an anti pattern to tag objects types. (Ie. the `MoveType` is redundant when each move already has a specific message type). That said, this is not a static problem / project, so probably fine this way.